crates/burn-dataset/src/vision/image_folder.rs

```diff
@@ -401,6 +479,36 @@ impl ImageFolderDataset {
         Self::with_items(items, classes)
     }
 
+    /// Create an image segmentation dataset with the specified items.
+    ///
+    /// # Arguments
+    ///
+    /// * `items` - List of dataset items, each item represented by a tuple `(image path, labels)`.
```

Comment:
> each item represented by a tuple `(image path, labels)`

To be a bit more precise, it should say "annotation path" (or "segmentation mask path"?) instead of "labels".

Reply:
fixed in the next commit.